ros2bag/ros2bag/verb/record.py

```diff
@@ -52,6 +51,13 @@ def add_arguments(self, parser, cli_name):  # noqa: D102
             help='time in ms to wait between querying available topics for recording. It has no '
              'effect if --no-discovery is enabled.'
         )
+        parser.add_argument(
+            '-b', '--bag-size', type=int, default=sys.maxsize,
+            help='provides a threshold for recording bagfile size in bytes, before the recording is split and '
+              'rolled over to a new bagfile. This value is only used as a guideline because actually enforcing a '
+              'maximum file size is difficult. The actual bagfile size may be a little over this value. '
+              'Defaults to system maximum integer value.'
```

Comment:
`Defaults to system maximum integer value.` is not understandable by users - could we make it "By default, all data are stored in a single file and this feature is disabled."